diff --git a/node_modules/@abp/ng.identity/fesm2022/abp-ng.identity.mjs b/node_modules/@abp/ng.identity/fesm2022/abp-ng.identity.mjs
index 6671110..3ef28dc 100644
--- a/node_modules/@abp/ng.identity/fesm2022/abp-ng.identity.mjs
+++ b/node_modules/@abp/ng.identity/fesm2022/abp-ng.identity.mjs
@@ -461,7 +461,7 @@ const DEFAULT_USERS_CREATE_FORM_PROPS = FormProp.createMany([
         displayName: 'AbpIdentity::DisplayName:LockoutEnabled',
         id: 'lockout-checkbox',
         defaultValue: true,
-    },
+    }
 ]);
 const DEFAULT_USERS_EDIT_FORM_PROPS = DEFAULT_USERS_CREATE_FORM_PROPS.map(prop => {
     if (prop.name === 'password') {
diff --git a/node_modules/@abp/ng.identity/proxy/lib/proxy/identity/models.d.ts b/node_modules/@abp/ng.identity/proxy/lib/proxy/identity/models.d.ts
index eadf983..c43a440 100644
--- a/node_modules/@abp/ng.identity/proxy/lib/proxy/identity/models.d.ts
+++ b/node_modules/@abp/ng.identity/proxy/lib/proxy/identity/models.d.ts
@@ -34,6 +34,7 @@ export interface IdentityUserCreateOrUpdateDtoBase extends ExtensibleObject {
     isActive: boolean;
     lockoutEnabled: boolean;
     roleNames: string[];
+    remark?: string;
 }
 export interface IdentityUserDto extends ExtensibleFullAuditedEntityDto<string> {
     tenantId?: string;
